docs/content/modeling/token-claims-contextual-tuples.mdx

```diff
@@ -0,0 +1,198 @@
+---
+sidebar_position: 9
+slug: /modeling/token-claims-contextual-tuples
+description: Using identity token claims to define contextual relations
+---
+
+import {
+  AuthzModelSnippetViewer,
+  CardBox,
+  CheckRequestViewer,
+  DocumentationNotice,
+  Playground,
+  ProductConcept,
+  ProductName,
+  ProductNameFormat,
+  RelatedSection,
+  RelationshipTuplesViewer,
+  UpdateProductNameInLinks,
+  WriteRequestViewer,
+} from '@components/Docs';
+
+# Using token claims as Contextual Tuples
+
+<DocumentationNotice />
+
+Contextual Tuples are used where an authorization check depends on certain dynamic or contextual relationships that have not been written to the <ProductName format={ProductNameFormat.ShortForm}/> store.
+
+Conceptually, they can be used to define Attribute Based Access Control (ABAC) policies based on relationships that are contextual to a request. To define ABAC policies that rely on specific attributes and conditions you can also use {ProductNameFormat.ShortForm}/>`s [conditions](./conditions.mdx).
```

Comment:
I'm not sure about this sentence, as it puts contextual tuples in the ABAC light, but then the article goes on to describe an alternate use-case not related to ABAC.

Also I would stay away from saying folks are defining ABAC policies, and just say they're enabling some ABAC use-cases

```
The ability to specify relationships that are contextual to a request enables some Attribute Based Access Control (ABAC) use-cases. To enable even more ABAC use-cases that rely on specific attributes and conditions you can also use {ProductNameFormat.ShortForm}/>`s [conditions](./conditions.mdx).
```